boot.php

```diff
@@ -201,6 +201,7 @@
 define('PRIORITY_MEDIUM',     30);
 define('PRIORITY_LOW',        40);
 define('PRIORITY_NEGLIGIBLE', 50);
+define('PRIORITIES', [PRIORITY_CRITICAL, PRIORITY_HIGH, PRIORITY_MEDIUM, PRIORITY_LOW, PRIORITY_NEGLIGIBLE]);
```

Comment:
I'd rather you moved all the priority constants in the `Core\Worker` class instead of adding global constants in `boot.php` that we're still trying to eliminate.

Reply:
Since we are in the RC phase, I wanted to make only changes with a light impact.

Reply:
Fair enough, please put this in your to-do list, adding a line to boot.php makes me cry.

Reply:
I feel with you :-)